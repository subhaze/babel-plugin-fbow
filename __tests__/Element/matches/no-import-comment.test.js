const babel = require("babel-core");
const plugin = require("../../../index.js");

const testCode = `
// @no-fbow
const a = document.querySelector("a");
const div = document.querySelector("div");
a.matches("div");
div.matches("p");
`;

it("adds import for matches once", () => {
	const { code } = babel.transform(testCode, {
		plugins: [plugin],
	});
	expect(code).toMatchSnapshot();
});
