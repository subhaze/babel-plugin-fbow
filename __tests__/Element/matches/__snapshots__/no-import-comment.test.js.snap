// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adds import for matches once 1`] = `
"
// @no-fbow
const a = document.querySelector(\\"a\\");
const div = document.querySelector(\\"div\\");
a.matches(\\"div\\");
div.matches(\\"p\\");"
`;
